source/includes/_tipser-elements.md

```diff
@@ -132,8 +132,8 @@ To display `Product` in a compact view, add the `data-tipser-view="compact"` att
 
 
 ## `Collection` Element
-
-You can render your collections as `Store`. Each collection, however, can be rendered separately as a simple product group, depending on your needs. It is all possible through the `Collection` Element.
+ 
+`Collection` is rendered as a simple product group.
```

Comment:
Maybe something along the lines of: "Collection is a group of simple product tiles. Clicking on any title displays a product dialog with more product details and add to cart button".

Also, later on it would be worth to mention that the products displayed in a collection are defined in the POS portal / tipser tools. (Just reading this section is not clear where do the products come from)